### PR TITLE
Issue #DP-1064 fix: fixed count issue and commented add from library button

### PR DESCRIPTION
**Pull request by santoshilimi**
…button

### CODE REVIEW

File: src/app/client/src/app/modules/shared-feature/components/textbook-list/textbook-list.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="text-capitalize fs-0-785 line-height-1-3 sb-pageSection-document py-5 ui grid mt-10">
 	<div class="twelve wide tablet five wide computer column pt-30">
 		<div class="sb-color-gray-400 d-inline-block br-2 pr-10 pt-5"><span class="fs-1-286 font-weight-bold">{{ contentStatusCounts?.sourcingOrgStatus?.total || 0 }}</span> {{resourceService.frmelmnts.lbl.total}}</div>
-		<div class="sb-color-warning d-inline-block br-2 px-10"><span class="fs-1-286 font-weight-bold">{{ contentStatusCounts?.sourcingOrgStatus?.pending + contentStatusCounts?.mvcContentCount || 0 }}</span> {{resourceService.frmelmnts.lbl.approvalPending}}</div>
+		<div class="sb-color-warning d-inline-block br-2 px-10"><span class="fs-1-286 font-weight-bold">{{ contentStatusCounts?.sourcingOrgStatus?.pending  || 0 }}</span> {{resourceService.frmelmnts.lbl.approvalPending}}</div>
```
Comment (venkateshwarans) [inline src/app/client/src/app/modules/shared-feature/components/textbook-list/textbook-list.component.html:4]: We need this count `contentStatusCounts?.mvcContentCount` no? If we enable this feature?

File: src/app/client/src/app/modules/contribute/components/program/program.component.html [hunk 2]
```diff
@@ -202,7 +202,7 @@ <h5 class="m-0 p-0 font-weight-bold d-inline-block">{{ programDetails?.name }} <
 												<div class="text-capitalize fs-0-785">
 													<span class="sb-color-primary d-inline-block br-2 pr-5">{{contributorTextbook.reviewCount || 0}}  {{resourceService.frmelmnts.lbl.pending}}</span>
 													<span class="sb-color-error d-inline-block br-2 px-5">{{contributorTextbook.rejectedCount || 0}} {{resourceService.frmelmnts.lbl.notAccepted}}</span>
-													<span class="sb-color-warning d-inline-block px-5">{{contributorTextbook.liveCount + contributorTextbook.mvcContentCount || 0}} {{resourceService.frmelmnts.lbl.sentForApproval}}</span>
+													<span class="sb-color-warning d-inline-block px-5">{{contributorTextbook.liveCount + contributorTextbook.mvcContributionsCount || 0}} {{resourceService.frmelmnts.lbl.sentForApproval}}</span>
```
Comment (venkateshwarans) [inline src/app/client/src/app/modules/contribute/components/program/program.component.html:205]: Won't this label be visible for `mvcContributionsCount` if `mvcLibraryFeature` is set to `disable`?

File: src/app/client/src/app/modules/shared-feature/components/textbook-list/textbook-list.component.html (hunk 1 above)
Comment (santoshilimi) [reply src/app/client/src/app/modules/shared-feature/components/textbook-list/textbook-list.component.html:4]: I am adding  mvcContributionsCount to sourcingOrgStatus?.pending in collection-hierarchy.service file if we enable this feature it works fine

File: src/app/client/src/app/modules/contribute/components/program/program.component.html (hunk 2 above)
Comment (santoshilimi) [reply src/app/client/src/app/modules/contribute/components/program/program.component.html:205]: mvcContributionsCount will be set to 0 if we don't have any mvc contents

**Review by venkateshwarans — APPROVED** [review]

**Pull request merged**